src/OrchardCore.Modules/OrchardCore.Forms/Entities/Scripting/ValidateRuleMethod.cs

```diff
@@ -0,0 +1,144 @@
+using System;

```

Comment:
Can you include a recipe (just attach under the PR) that contains a demonstration of how to use this, with a form?

Reply:
https://gist.github.com/SpikeXy/8d9dc39fa3cde80076e5f9883c3e43f0, the workflow does not created when the recipe inserted, maybe something wrong, so if we want to test methods in ValidateRuleMethod.cs we need to create the workflow manually.

Reply:
Thank you!